test/vfd.c

```diff
@@ -2271,46 +2271,51 @@ test_ros3(void)
 static int
 compare_splitter_config_info(hid_t fapl_id, H5FD_splitter_vfd_config_t *info)
 {
-    int                        ret_value = 0;
-    H5FD_splitter_vfd_config_t fetched_info;
+    int                         ret_value    = 0;
```

Comment:
This looks like the reverse of what we should be doing.  (i.e. 1.12 is better code than develop)

Reply:
Unfortunately, 1.12 is the only branch where the splitter test fail. 1.10 was the same until changes were merged from develop to 1.10 for the 1.10.7 release.
Until I get all the other changes through, I commented out the test so I don't create one huge PR.